### PR TITLE
Desktop: Hide completed todos from search results

### PR DESCRIPTION
**Pull request by arpitg24**
Hides checked todos from search results when Show completed todos is unchecked. It is a simple conditional rendering if else statement which doesnt render completed todos when the concerned setting is unchecked. Solves #4581

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -63,7 +63,7 @@
     "husky": "^3.0.2",
     "lerna": "^3.22.1",
     "lint-staged": "^9.2.1",
-    "madge": "^4.0.0",
+    "madge": "^4.0.2",
```
Comment (roman-r-m) [inline package.json:66]: Is this really needed? madge seems to be dev tool to generate a graph of dependencies

Comment (roman-r-m) [reply package.json:66]: Same for all `package-lock.json` changes

Comment (arpitg24) [reply package.json:66]: I didn't intentionally do it maybe npm upgraded it automatically when I compiled the code.

Comment (roman-r-m) [reply package.json:66]: Most likely, yes, but please check what you're commiting

**Comment by roman-r-m** [comment]
I'm not sure if that's the best place to implement this (but wait for Laurent's before rushing to make any changes).

Sure it gets the job done but maybe it's better to filter at the place where the search is performed? This way you don't have to pass around all checked todos only for them to be filtered before being displayed.

Plus implementing it as part of search service means it'll work not only on desktop but also on CLI and mobile, I think.

**Comment by laurent22** [comment]
It would be good to have this feature but I agree with Roman, I feel this is a hack and we want to avoid this. Indeed it should be done at the service level, or wherever the data is fetched so that it works on all platforms. Doing it that way also means we can unit test the change.

Because of this, I would prefer to close the pull request for now, but feel free to create a new one if you want to give it another try by modifying the service.

File: package.json (hunk 1 above)
Comment (arpitg24) [reply package.json:66]: Is it advised for me to close this PR and create a new one in which only NoteList.tsx is changed?

**Pull request closed without merging**